frontend/src/contexts/PromotionsListContext.tsx

```diff
@@ -1,90 +0,0 @@
-import React, {
```

Comment:
Just moved things out of this file and into `frontend/src/contexts/PromotionsList.tsx`